sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -204,6 +215,8 @@ class CacheManager extends Logging with AdaptiveSparkPlanHelper {
       cachedData = cachedData.filterNot(cd => plansToUncache.exists(_ eq cd))
     }
     plansToUncache.foreach { _.cachedRepresentation.cacheBuilder.clearCache(blocking) }
+    CacheManager.logCacheOperation(s"Removed ${plansToUncache.size} Dataframe cache " +
```

Comment:
I think we need to migrate to the structured logging API here.
Framework: https://github.com/apache/spark/pull/45729
Example migration PR: https://github.com/apache/spark/pull/45834